asmcomp/cmm.ml

```diff
@@ -94,7 +94,15 @@ let negate_float_comparison = Lambda.negate_float_comparison
 let swap_float_comparison = Lambda.swap_float_comparison
 type label = int
 
-let label_counter = ref 99
+let init_label = 99
+
+let label_counter = ref init_label
+
+let set_label l =
```

Comment:
The function name `set_label`  goes along with existing `new_label` and `cur_label` all of which set label_counter. The name `label_counter` is not exposed at all in the interface.  I changed the type of `set_label` from `int` to `label` to make it more descriptive and better aligned with the other two functions. Is it enough or do you think it would be better to rename this function anyway?

Reply:
This seems ok.